felix/dataplane/linux/bpf_maps.go

```diff
@@ -0,0 +1,81 @@
+//go:build !windows
```

Comment:
Could this go to https://github.com/projectcalico/calico/blob/master/felix/bpf/bpfutils ? Would that also avoid the circular import? I think so.

Reply:
Moving to ```bpfutils``` still results in circular dependency